experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/node.ts

```diff
@@ -14,11 +14,11 @@ import { ProxyNode, SharedTreeNode } from "./types";
  * Provides various functions for analyzing {@link SharedTreeNode}s.
  * @alpha
  * @privateRemarks
- * Inlining the typing of this interface onto the `node` object provides slightly different .d.ts generation,
+ * Inlining the typing of this interface onto the `Tree` object provides slightly different .d.ts generation,
  * which avoids typescript expanding the type of TreeNodeSchema and thus encountering
  * https://github.com/microsoft/rushstack/issues/1958.
  */
-export interface NodeApi {
+export interface TreeApi {
```

Comment:
Good question. My thought was that if somebody ever did want to use the type of `Tree` explicitly, it'd be more consistent and less confusing to have it called `TreeApi`, since they're both called "Tree". It's also possible that we'll put "global tree APIs" on here - for example, things that go through the context under the hood (rather than making the user do `Tree.context(foo).something`). So, I think it's appropriate.

Reply:
I misinterpreted what this was. Looks good.